marconi-chain-index/src/Marconi/ChainIndex/Extract/Datum.hs

```diff
@@ -0,0 +1,105 @@
+{-# LANGUAGE DerivingStrategies #-}
```

Comment:
I think this module should only contain datum extraction functions from transactions. The `DatumRow` and `createTable` should maybe not be here?

Reply:
Right, so maybe it doesn't look like it belongs to an extract module, but as the table has the same function between Utxo indexer and AddressDatum then then we could reuse it, no? (Maybe call the module Marconi.ChainIndex.Datum?)